universum/analyzers/utils.py

```diff
@@ -1,32 +1,64 @@
 import json
 import sys
+import argparse
+import glob
+import subprocess
 
+from typing import Callable, List
+from typing_extensions import TypedDict
 
-def add_files_argument(parser):
-    parser.add_argument("--files", dest="file_list", nargs='+', required=True,
-                        help="Python files and Python packages for analyzer. "
-                             "Files could be defined as a single python file"
-                             " *.py or directories with __init__.py file in the directory.")
+ReportData = TypedDict('ReportData', {'path': str, 'message': str, 'symbol': str, 'line': int})
 
 
-def add_result_file_argument(parser):
+def run_parse_report(cmd: List[str], parse: Callable[[str], List[ReportData]], out_file: str) -> int:
```

Comment:
The name makes an impression that the function runs a report.
I suggest to rename it like "run_and_parse_report".
I also suggest to rename "parse" argument to "parse_function".